rules/do-you-create-a-sprint-review-retro-email/rule.md

```diff
@@ -31,7 +31,7 @@ Firstly, create a new email copying the information from the previous Sprint Rev
 |          |     |
 | -------- | --- |
 | To:      | {{Product Owner}} |
-| Cc:      | {{Sprint Review Attendees}}, {{Sprint Review Reporting Email}} |
+| Cc:      | {{Sprint Review Attendees}}, {{Sprint Review Reporting Email}} e.g. SSWSprintReviews@sswcom.onmicrosoft.com|
```

Comment:
change from
{{Sprint Review Reporting Email}} e.g. SSWSprintReviews@sswcom.onmicrosoft.com
to
{{Sprint Review Reporting Email e.g. SSWSprintReviews@sswcom.onmicrosoft.com}}